src/nats.h

```diff
@@ -4564,6 +4564,28 @@ natsSubscription_AutoUnsubscribe(natsSubscription *sub, int max);
 NATS_EXTERN natsStatus
 natsSubscription_QueuedMsgs(natsSubscription *sub, uint64_t *queuedMsgs);
 
+/** \brief Gets the subscription id.
+ *
+ * Returns the id of the given subscription.
+ *
+ * @param sub the pointer to the #natsSubscription object.
+ * @param id if not `NULL`, the memory location where to store the subscription id.
```

Comment:
But now that we make individual getters, it is a bit silly to support `NULL` for the return location. That is, why would one call this function without wanting to retrieve the id?